build.gradle

```diff
@@ -386,6 +386,7 @@ checkstyle {
 }
 
 tasks.withType(Checkstyle) {
+    dependsOn(':licenseHeaders')
```

Comment:
As long as one of the required checks runs this task then it will run with every PR. I thought it would make sense to run it as part of the checkstyle tasks.